docs/source/dataframe-best-practices.rst

```diff
@@ -216,28 +216,25 @@ operation:
 
 For more information see :doc:`Joins <dataframe-joins>`.
 
-Store Data in Apache Parquet Format
------------------------------------
+Use Parquet
+-----------
 
-`Apache Parquet <https://parquet.apache.org/>`_ is a
-columnar binary format that is easy to split into multiple files (easier for
-parallel loading). It is also a common format used by other big data
-systems like `Apache Spark <https://spark.apache.org/>`_ and `Apache Impala
-<https://impala.apache.org/>`_, and so it is useful to interchange with other
-systems:
+`Apache Parquet <https://parquet.apache.org/>`_ is a columnar binary format.
+It is the de-facto standard for the storage of large volumes of tabular data
+and our recommended storage solution for basic tabular data.
 
 .. code-block:: python
 
    df.to_parquet('path/to/my-results/')
    df = dd.read_parquet('path/to/my-results/')
 
-Dask supports reading parquet files with different engine implementations of
-the Apache Parquet format for Python:
-
-.. code-block:: python
-
-   df1 = dd.read_parquet('path/to/my-results/', engine='fastparquet')
-   df2 = dd.read_parquet('path/to/my-results/', engine='pyarrow')
+When compared to formats like CSV, Parquet brings the following advantages:
 
+1. It's faster to read and write, often by 4-10x
+2. It's more compact to store, often by 2-5x
```

Comment:
Where are these numbers coming from?

Reply:
My brain mostly

Reply:
I'm not a huge fan of this as it's not clear that these numbers are correct / up-to-date. It's not a huge deal as we say these are "often" the case -- not always. But still, it'd be nice to see some sort of testing (even just a `%timeit` ipython cell) before we put numbers in the docs.

Reply:
That's a good point, a quick inspection with `%timeit` shows it's closer to ~10-30x faster to read/write:
```python
In [3]: df = dask.datasets.timeseries().compute()

In [4]: %timeit df.to_parquet("ts.parquet")
407 ms ± 11.7 ms per loop (mean ± std. dev. of 7 runs, 1 loop each)

In [5]: %timeit df.to_csv("ts.csv")
13.7 s ± 1.06 s per loop (mean ± std. dev. of 7 runs, 1 loop each)

In [6]: df = df.reset_index(drop=True)

In [7]: %timeit df.to_parquet("ts.parquet")
290 ms ± 12.7 ms per loop (mean ± std. dev. of 7 runs, 1 loop each)

In [8]: %timeit df.to_csv("ts.csv")
11.5 s ± 529 ms per loop (mean ± std. dev. of 7 runs, 1 loop each)

In [9]: %timeit pandas.read_csv("ts.csv")
1.3 s ± 37.6 ms per loop (mean ± std. dev. of 7 runs, 1 loop each)

In [10]: %timeit pandas.read_parquet("ts.parquet")
138 ms ± 19.6 ms per loop (mean ± std. dev. of 7 runs, 1 loop each)

In [11]: df = dask.datasets.timeseries().compute()

In [12]: df.to_parquet("ts.parquet")

In [13]: df.to_csv("ts.csv")

In [14]: %timeit pandas.read_parquet("ts.parquet")
162 ms ± 7.67 ms per loop (mean ± std. dev. of 7 runs, 1 loop each)

In [15]: %timeit pandas.read_csv("ts.csv")
2.49 s ± 195 ms per loop (mean ± std. dev. of 7 runs, 1 loop each)

```